nautobot/ui/src/styles/shared.css

```diff
@@ -0,0 +1,4 @@
+.nautobot-add-form>.form-group>.chakra-form-control>.css-qq8aff>.css-0 {
```

Comment:
I strongly suspect that this is fragile, not sure where `qq8aff` is coming from but I feel like it might change at any time.